mentoring/tests/integration/test_answer.py

```diff
@@ -47,37 +47,40 @@ def test_answer_edit(self):
         self.assertEqual(header1.text, 'XBlock: Answer Edit 1')
 
         # Check <html> child
-        p = mentoring.find_element_by_css_selector('div.xblock > p')
+        p = mentoring.find_element_by_css_selector('p')
```

Comment:
As a reminder, these tests are almost certainly going to have to be refactored in order to be accepted by upstream. They will need to use bok_choy page objects.